content/gettingstarted/index.adoc

```diff
@@ -39,88 +39,67 @@ icon:lightbulb[size=1x]{nbsp}For several commands listed on this page, the OpenS
 
 == Installing Kiali
 
-=== Preparation
+Kiali can be installed in an OpenShift or Kubernetes cluster environment. For OpenShift only, a minimal preparation is necessary.
```

Comment:
line 46 says "on Openshift" and here it is "in Openshift"

Reply:
I'm not sure, but maybe `on OpenShift` and `in an OpenShift cluster environment` are both correct?
Maybe I can simplify here and just write `Kiali can be installed on OpenShift or Kubernetes.` wdyt?